src/main/java/net/minestom/server/entity/Player.java

```diff
@@ -263,10 +264,11 @@ public Player(@NotNull UUID uuid, @NotNull String username, @NotNull PlayerConne
     }
 
     @ApiStatus.Internal
-    public void setPendingInstance(@NotNull Instance pendingInstance) {
```

Comment:
maybe create a second method without the hardcore parameter to keep backwards compability

Reply:
It is an internal method, what is the point of keeping compatibility?